Source/WebInspectorUI/UserInterface/Views/Variables.css

```diff
@@ -408,6 +408,11 @@ body:is(.mac-platform.monterey, .mac-platform.big-sur):not(.docked) {
         --slider-track-tick-background: hsl(0, 0%, 55%);
     }
 
+    @media (prefers-color-scheme: dark) and (prefers-contrast: more) {
```

Comment:
This is already wrapped inside `@media (prefers-color-scheme: dark)` so that'd be implicit here.

I think Devin's suggestion was if we were to avoid nesting altogether and have top level CSS (but I don't see a reason to not use nesting, I would rather have us adopt our own modern CSS features). I wouldn't be against refactoring most of our WebInspector CSS to use nesting fwiw.